src/context/App.tsx

```diff
@@ -37,7 +37,7 @@ export const defaultSettings: SettingsState = {
   markOnClick: false,
   openAtStartup: false,
   appearance: Appearance.SYSTEM,
-  colors: true,
+  colors: false,
```

Comment:
Maybe I'm missing something; I thought the intent of this PR was to keep colors enabled by default?